lib/active_model/serializer/reflection.rb

```diff
@@ -119,6 +119,7 @@ def build_association(subject, parent_serializer_options)
               serializer_options(subject, parent_serializer_options, reflection_options)
             )
           rescue ActiveModel::Serializer::CollectionSerializer::NoSerializerError
+            warn "No serializer found for #{association_value}."
```

Comment:
Reason for it is fine, if it can cause inconsistent serialization.  

We use `warn` elsewhere and not `STDERR`, so that's a good choice.

Maybe also `AMS.logger.warn { "No serializer found for #{association_value}." }` ?